sbin/prepareWorkspace.sh

```diff
@@ -37,14 +36,55 @@ ALSA_LIB_CHECKSUM=${ALSA_LIB_CHECKSUM:-5f2cd274b272cae0d0d111e8a9e363f0878332915
 ALSA_LIB_GPGKEYID=${ALSA_LIB_GPGKEYID:-A6E59C91}
 FREETYPE_FONT_SHARED_OBJECT_FILENAME="libfreetype.so*"
 
+
+copyFromDir() {
+  echo "Copyng existing ${BUILD_CONFIG[OPENJDK_FOREST_DIR_ABSPATH]} to `pwd`/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]} to be built"
+  # we really do not want to use .git for dirs, as we expect user have them set up, ignoring them
+  local files=$(find ${BUILD_CONFIG[OPENJDK_FOREST_DIR_ABSPATH]} -maxdepth 1 -mindepth 1 | grep -v -e "/workspace$" -e "/build$" -e "/.git" -e -"/build/")
```

Comment:
Not sure about the ignores for workspace, build, .git, ...., this list could grow or become wrong...
I think we should assume the source tarball is exactly that, if someone puts something in their that is "wrong" then we are not really judging if it is right or wrong....?

Reply:
Teh source tarball is most likely ok. But in case of local directory? That will always contain at least .git. And in many cases also the workspace and build:(